README.md

```diff
@@ -11,4 +11,8 @@ To start the services, run `docker-compose up -d`.
 TODO 
 
 Steps to generate a topology from the archetypes
-Launch a topology + run it on the cluster
+Launch a topology ( (1) via Flux (2) via Java) + run it on the cluster
```

Comment:
The Java topologies are not recommended - this repo is meant as a basis for tutorials and workshops and we'll be using Flux